package.json

```diff
@@ -63,9 +63,10 @@
     "aegir": "^10.0.0",
     "chai": "^3.5.0",
     "eslint-plugin-react": "^6.9.0",
+    "go-ipfs-dep": "^0.4.5",
```

Comment:
@diasdavid also note that this PR pulls in go-ipfs 0.4.5 as ipfsd-ctl doesn't have it yet. We need to remove this line once we've updated the ecosystem modules to 0.4.5.